k-distribution/tests/regression-new/checkWarns/Makefile

```diff
@@ -1,4 +1,4 @@
 KOMPILE_FLAGS=-w2e
-KOMPILE_BACKEND=llvm
+KOMPILE_BACKEND=haskell
```

Comment:
Hmmmm, why is this here? Is there a test in this directory which has an existential on the RHS (so it was now failing for the wrong reason?)

Maybe we shouldn't be failing until we collect _all_ the warnings then.

Reply:
Or maybe we should be running these checks tests with both backends all the time.

It makes me nervous that we'll be missing some warnings/errors because we're going through the slightly different `kompile` pipelines.